Donation-platform/src/main/webapp/trendingScript.js

```diff
@@ -55,6 +58,21 @@ function updateResultsOnPage(charities) {
   // add results to the page
   //for (const name of charities) {
   for (index = 0; index < charities.length; index++) {
-    resultsContainer.innerHTML += '<li>' + charities[index] + '</li>';
+    resultsContainer.innerHTML += '<li>' + charities[index][0] + ': ' + charities[index][1] + '</li>';
+
   }
+}
+
+/**
+ * Updates the display with bootstrap cards. (IN PROGRESS)
+ */
+function updateCardsOnPage(charities) {
+    const cards = document.getElementById('cards');
+
+    cards.innerHTML = '';
+
+    for (index = 0; index < charities.length - 1; index += 2) {
```

Comment:
IF you put charity object in the above, you can simplify this loop condition

Reply:
yes, changed it to a forEach over the JSON itself

Reply:
But why do i Still see you iterating over 2 elements?

Reply:
that's the cards function that I have now changed, but is meant for the more advanced cards display that i was working with on the side. For this PR, my goal was to get a very minimal display working.

Reply:
discussed offline, will be removed in next PR.